src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -215,6 +215,10 @@ def save(self, args, kwargs):
 
     def result(self):
         model_cls = self._cmd.get_models(self._model_type)
+        # todo: REmove temp work around!!!
+        if model_cls is None:
+            from azure.mgmt.imagebuilder.models import ImageTemplate
```

Comment:
@tjprescott  this was my very weird work around.... I will definitely remove this when I get your fixes.